K_Scaffold/Template v2/scaffold/tabs/_tabs.pug

```diff
@@ -33,27 +57,27 @@ mixin tabs(name="tabs",defaultActiveTabIndex=0)
       - {string} [tabName] - The name of the tab. Defaults to `tab` followed by a index count starting at 1 (so tab1, then tab2, ...)
       - {object} [button] - object passed to the +action mixin to build the action button for switching tabe. By default a trigger is defined, but you can override it.
       - {string} [container] - tag to use to contain the whole tab. A div by default but can be overridden.
-  mixin tab(tabName,button={},container="div")
+  mixin tab({tabName,button={},container="div"})
     - tabName = actionButtonName(replaceProblems(tabName || `tab${tabs.length + 1}`));
     - if (tabs.filter(tab => tab.name === tabName).length) { throw new Error(`Tab "${tabName}" already exists in "${name}".`); }
     //- Cleanup the name of the navigation button
     - button.name = `nav-tabs-${name}--${tabName}`;
     //- Cleanup the class, add our own internal classes
     - button.class = button.class ? replaceProblems(button.class) : "";
-    - button.class = `tabs__button tabs--${name}__button tabs--${name}__button--${tabName} ${button.class}`;
-    - button.class = button.class + (tabs.length === defaultActiveTabIndex ? " k-active-tab" : "");
+    - button.class = `tabs__button tabs--${name}__button tabs--${name}__button--${tabName}${!persistent && defaultActiveTab === tabName ? 'k-active-button' : ''} ${button.class}`;
```

Comment:
I think there is a missing space here between the `tabs--${name}__button--${tabName}` and the possible `k-active-button`.
I also think that having the optional "active" class that will be moved around by jQuery at the end makes it easier to follow in the inspect mode mode of browser.

So I suggest 
```javascript
button.class = `tabs__button tabs--${name}__button tabs--${name}__button--${tabName} ${button.class}${!persistent && defaultActiveTab === tabName ? ' k-active-button' : ''}`;
```

Reply:
Makes sense. Probably want a ternary on the `button.class` so we don't wind up with `undefined` in the class list

Reply:
There is one the line before already :)

Reply:
Gah, so there is!